pkgs/applications/editors/emacs-modes/telega/libtgvoip.nix

```diff
@@ -0,0 +1,34 @@
+{
+pkgs
+, clangStdenv
+, libopus
+, openssl
+, libpulseaudio
+, alsaLib
+, pkgconfig
+, writeText
+, llvmPackages
+, libtool
+, writeShellScriptBin
+}:
+clangStdenv.mkDerivation {
+
+  name = "libtgvoip";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "zevlg";
+    repo = "libtgvoip";
+    rev = "8e08116896bb6186d2eefb636724933f7dd8bcea";
+    sha256 = "00qxh8vd1sag4hyq99jxfjnqgrf788pc050s0kl08ma48dm8hi5m";
+  };
+
+  preConfigure = ''
+      export AM_LIBTOOLFLAGS=" --tag CXX "
+    '';
+
+  enableMultiThreaded = true;
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ libopus openssl libpulseaudio alsaLib ];
+
```

Comment:
Missing `meta` attrs.

Reply:
> Missing `meta` attrs.

Actually I was just thinking that whole this library is telega specific (and certainly not top-level), so added no meta, and no version.